core/src/test/java/org/apache/iceberg/TestManifestReaderStats.java

```diff
@@ -82,6 +88,13 @@ public void testReadWithFilterIncludesFullStats() throws IOException {
       ManifestEntry<DataFile> entry = entries.iterator().next();
       assertFullStats(entry.file());
     }
+
+    try (ManifestReader<DataFile> reader = ManifestFiles.read(manifest, FILE_IO)
+        .filterRows(Expressions.notStartsWith("data", "Z"))) {
+      CloseableIterable<ManifestEntry<DataFile>> entries = reader.entries();
+      ManifestEntry<DataFile> entry = entries.iterator().next();
+      assertFullStats(entry.file());
+    }
```

Comment:
I think this is testing something else, so this file doesn't need to change. Unless I'm missing something.

Reply:
You're right. I think this is one of the cases where I added in several tests in order to better understand how everything was executed (occasionally hopping through with the debugger). I'll remove the changes to this file.